### ISSUE TITLE
Some timezone transitions are wrong

### ISSUE THREAD

**Issue by davidje13** [Jimbly/timezone-mock#19]
It appears that the list of timezone switch dates in this project go quite a way into the future, but laws have changed since it was written and those switches are no-longer correct (even for current years). This makes using the library for testing more painful than it needs to be.

I suggest limiting the data to past-and-present, and have no attempt to predict future switches. Then document which years are best for testing, and document the exact dates and times when those switches happen. For example, through some investigation I found that the US/Pacific timezone is accurate for 2006, with the following transitions:

```
2006-04-02 2:00 PDT [10:00 UTC timestamp 1143972000] - PDT -> PST
2006-10-29 2:00 PST [09:00 UTC timestamp 1162112400] - PST -> PDT
```

So I can just mock the system time to around 2006 and test. (2006 was particularly easy to validate because Wikipedia happens to call it out explicitly)

I have a similar problem with Brazil/East, which also seems to have changed its laws and is harder to find data on. A quick note in the readme would really help!

**Comment by Jimbly** [comment]
Thanks for the info!  I pulled the data from Linux's tz_data, which I suspect is updated very quickly after law changes, I'll look into automating pulling that data regularly, and also add a note to the docs after I updated it with at least current data - the data in there is probably from 4 years ago.

**Comment by davidje13** [comment]
For repeatability of tests, I think you should encourage using past (known-good) data. Worst outcome is that tests get written against predicted future transitions, then break when the rule changes.

Maybe that just means putting an automated note in the documentation along the lines of "timezone data is correct up to <date of import>; behaviour after this time may change in later releases to reflect updated laws".

**Comment by Jimbly** [comment]
Yeah, good point.  I sometimes forget that this module has no actual use besides writing tests =).

**Comment by Jimbly** [comment]
Data updated (I see Brazil/East did in fact change it's transition times), note added to README, and published as v1.0.4.  Thanks!
